src/pixbuf.rs

```diff
@@ -178,10 +163,44 @@ impl Pixbuf {
         }
     }
 
-    pub fn copy(&self) -> Pixbuf {
```

Comment:
Here it also was not returning an Option.

Maybe take a diff of the API changes here to make sure the API is not regressing without noticing.